test/db.test.js

```diff
@@ -399,11 +399,13 @@ describe('Database', function () {
       process.removeAllListeners('uncaughtException');
 
       process.on('uncaughtException', function MINE (ex) {
+        process.removeListener('uncaughtException', MINE);
```

Comment:
This listener not being removed was causing me a lot of grief when trying to debug other test failures.  :confounded: